otsdaq-mu2e/FEInterfaces/DTCFrontEndInterface.h

```diff
@@ -202,6 +202,9 @@ class DTCFrontEndInterface : public CFOandDTCCoreVInterface
 	void 								BufferTest_detached					(__ARGS__);
 
 
+        void                                                            SoftwareDR                                     (__ARGS__);
```

Comment:
I think we should do our best to avoid acronyms to improve the chance future code readers can understand things quickly. If you agree, can you please change the name of this function to SoftwareDataRequest?

Reply:
done